main.tf

```diff
@@ -211,11 +211,85 @@ resource "aws_autoscaling_group" "container_instance" {
   }
 }
 
+resource "aws_launch_configuration" "container_instance_scheduled_tasks" {
+  lifecycle {
+    create_before_destroy = true
+  }
+
+  root_block_device {
+    volume_type = "${var.root_block_device_type}"
+    volume_size = "${var.root_block_device_size}"
+  }
+
+  name_prefix          = "lc${title(var.environment)}ContainerInstanceScheduledTasks-"
```

Comment:
I'd use name rather than name_prefix because we shouldn't have any name collisions. That said, this shouldn't be an issue to use name_prefix. However, I don't think putting the environment in the name is useful given that each of our AWS accounts are isolated by environment already. I'm not against putting env in names of things, but we should be consistent. Is the plan to start putting environment in names of things?

Reply:
This was a copy-pasta. The `var.environment` here is specified as `ecs`, rather than develop/staging/production.

https://github.com/builtinx/infrastructure/blob/794e758b109e05bcb9ca90e5e7f122985cf6f151/aws/test/ecs.tf#L40